src/app/vault/view.component.ts

```diff
@@ -68,6 +72,14 @@ export class ViewComponent extends BaseViewComponent implements OnChanges {
 
     async ngOnChanges() {
         await super.load();
+
+        if (this.cipher.collectionIds?.length > 0) {
+            const collection = await this.collectionService.get(this.cipher.collectionIds[0]);
```

Comment:
Why do we take only the first collection?

Reply:
This was lazy code. This is because we can't have more that 1 collection for a cipher in cozy-pass.
But Bitwarden can, so lets make this code work for both.

New version : 
```ts
        if (this.cipher.collectionIds?.length > 0) {
            let readOnly = true;

            // in Bitwarden's protocol a cipher is writable if part of at least one writable collection
            for (const collectionId of this.cipher.collectionIds) {
                const collection = await this.collectionService.get(collectionId);
                if (!collection.readOnly) {
                    readOnly = false;
                    break;
                }
            }

            this.isReadOnly = readOnly;
        } else {
            this.isReadOnly = false;
        }
```